ui/packages/catalog-ui-search/src/main/webapp/react-component/filter/filter.js

```diff
@@ -82,7 +81,20 @@ class Filter extends React.Component {
   componentDidMount() {
     this.updateSuggestions()
   }
+  getListofSupportedAttributes = (settingsModel) => {
+    // if no source is selected and settingsModel is present from parent component we want to present all attributes as available 
+    if(settingsModel.length == 0){
```

Comment:
```suggestion
    if (settingsModel.length == 0) {
```
brought to you by `/projects/ddf/ui>> yarn format -w`